jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/resources/META-INF/CaseMgmtorm.xml

```diff
@@ -21,10 +21,12 @@
         log.processInstanceId, <!-- case top level process instance id -->
         log.outcome,
         log.slaDueDate,
-        log.slaCompliance   
+        log.slaCompliance,
+        parent.correlationKey
       )
       from
         ProcessInstanceLog log
+      left join ProcessInstanceLog parent on log.parentProcessInstanceId = parent.processInstanceId
```

Comment:
has someone tested this on large ProcessInstanceLog table? asking if it has significant impact on performance or maybe there will be a need for some indexes...

Reply:
EXPLAIN SELECT * FROM ProcessInstanceLog a LEFT JOIN ProcessInstanceLog b on a.id = b.parentProcessInstanceId 
throws this this data IDX_PInstLog_parentPInstId will be used

The only thing that come to my mind is to duplicate the call or something.

Reply:
so with this we assume it will be performant enough?

@MarianMacik do we have already some test cases with loaded db tables?

Reply:
@mswiderski @MarianMacik 
just tried a small snippet here to load a mysql database 
https://github.com/kiegroup/jbpm/pull/1562

2800 (1400 + 1 child per case) cases about to 3 seconds with or without the join. I let the code in there (SubCaseServiceImplLoadTest)

Reply:
ok, so we don't have much difference then we should be fine, thanks @elguardian